services/fxaccounts/FxAccountsWebChannel.jsm

```diff
@@ -244,6 +245,32 @@ FxAccountsWebChannel.prototype = {
           .login(data)
           .catch(error => this._sendError(error, message, sendingContext));
         break;
+      case COMMAND_OAUTH_LOGIN:
+        const rustFxABridge = fxAccounts._rustFxABridge;
+        rustFxABridge
+          .completeOAuthFlow(data.code, data.state)
+          .then(async () => {
+            const profile = await rustFxABridge.getProfile();
+            let state = await rustFxABridge.toJSON();
+            state = JSON.parse(state);
+            const data = {
+              email: profile.email,
+              services: {},
+              sessionToken: state.session_token,
+              uid: profile.uid,
+              verified: true,
+            }
+
+            return this._helpers.login(data)
+          })
+          .then(() => {
+            // redirect to the sms page to connect more devices
+            browser.loadURI(`${accountServer.spec}sms`, {
```

Comment:
need to check this, there might be a helper that takes care of this "success" page